src/components/filters/MeasureValueFilter/DropdownBody.tsx

```diff
@@ -189,28 +189,30 @@ class DropdownBodyWrapped extends React.PureComponent<IDropdownBodyProps, IDropd
         this.setState({ value: { ...this.state.value, to } });
     };
 
+    private round = (n: number): number => parseFloat(n.toFixed(6));
```

Comment:
Add comment here about that 6 decimal point numbers are supported by platform

Reply:
And it would be better to create `const MAX_DECIMAL_POINT_DIGITS = 6;` and add comment there